doc/source/tune-usage.rst

```diff
@@ -0,0 +1,318 @@
+Tune User Guide
+===============
+
+Tune Overview
+-------------
+
+.. image:: images/tune-api.svg
+
+Tune schedules a number of *trials* in a cluster. Each trial runs a user-defined Python function or class and is parameterized either by a *config* variation from Tune's Variant Generator or a user-specified **search algorithm**. The trials are scheduled and managed by a **trial scheduler**.
+
+More information about Tune's `search algorithms can be found here <tune-searchalg.html>`__.
+
+More information about Tune's `trial schedulers can be found here <tune-schedulers.html>`__.
+
+Tune provides a ``run_experiments`` function that generates and runs the trials as described by the `experiment specification <tune-usage.html#experiment-configuration>`__.
+
+.. autofunction:: ray.tune.run_experiments
+    :noindex:
+
+This function will report status on the command line until all Trials stop:
+
+::
+
+    == Status ==
+    Using FIFO scheduling algorithm.
+    Resources used: 4/8 CPUs, 0/0 GPUs
+    Result logdir: ~/ray_results/my_experiment
+     - train_func_0_lr=0.2,momentum=1:  RUNNING [pid=6778], 209 s, 20604 ts, 7.29 acc
+     - train_func_1_lr=0.4,momentum=1:  RUNNING [pid=6780], 208 s, 20522 ts, 53.1 acc
+     - train_func_2_lr=0.6,momentum=1:  TERMINATED [pid=6789], 21 s, 2190 ts, 100 acc
+     - train_func_3_lr=0.2,momentum=2:  RUNNING [pid=6791], 208 s, 41004 ts, 8.37 acc
+     - train_func_4_lr=0.4,momentum=2:  RUNNING [pid=6800], 209 s, 41204 ts, 70.1 acc
+     - train_func_5_lr=0.6,momentum=2:  TERMINATED [pid=6809], 10 s, 2164 ts, 100 acc
+
+Experiment Configuration
+------------------------
+
+.. contents::
+    :local:
+    :backlinks: none
+
+Specifying Experiments
+~~~~~~~~~~~~~~~~~~~~~~
+
+There are two ways to specify the configuration for an experiment - one via Python and one via JSON.
+
+The first way to specify a configuration is to create an Experiment object. You can then pass in either
+a single experiment or a list of experiments to `run_experiments`, as follows:
+
+.. code-block:: python
```

Comment:
I didn't find this section terribly easy to read. Instead of happing "API snippets", consider larger examples that can be copy pasted.